ui/app/css/itcss/components/request-signature.scss

```diff
@@ -61,9 +61,9 @@
   }
 
   &__header__text {
+    @include H3;
```

Comment:
Similar to the encrypt/decrypt changes but only appears on the request-signature of older signing types not present on our test dapp. Including only a screenshot of the after because if we decide to bump down font size to H4 on these it will be because of requesting a key not this:

<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/4448075/88584175-dfc04880-d016-11ea-97c5-12cd6ab0c901.png" />


</details>